src/components/visualization/vegaLiteViz/vegaLiteViz.tsx

```diff
@@ -53,11 +71,39 @@ function VegaLiteViz(props: VegaLiteData) {
     }
   }, [props.spec, isDarkTheme])
 
+  const handleDownload = (format: 'svg' | 'png') => {
+    if (downloadUrls) {
+      // Create a temporary anchor element
+      const link = document.createElement('a')
+      link.href = downloadUrls[format] || ''
+      link.download = `Vega-Lite-Visualization.${format}`
```

Comment:
consider adding support for href in menu.

Reply:
Could you elaborate which kind of support I should add?

Reply:
maybe you can tweak or update popover to support `anchor` tags